File: Ex3/Task3.2/YearsBeforeRetirement.scala
/**
 * Shows how many years I have left before retirement
 */
object YearsBeforeRetirement {
    def main(args: Array[String]) {
        var myAge: Int = 21
        var retirementAge: Int = 68
        var yearsLeft: Int = retirementAge - myAge

        println("My age now is " + myAge)
        println("I will retire at the age of " + retirementAge)
        println("Years left working is " + yearsLeft)
    }
}

File: ex3/task3.2/YearsBeforeRetirement.java
/**
* Prints 3 lines of text with 3 variables
* Doesn`t need to be tested
* @author Vlad Georgescu 
*/
public class YearsBeforeRetirement
{
    public static void main(String[] args)
    {
        int myAgeNow;
        int retirementAge;
        int yearsBeforeRetirement;
        myAgeNow = 20;
        retirementAge = 65;
        yearsBeforeRetirement = retirementAge - myAgeNow;
        System.out.println("I am " + myAgeNow);
        System.out.println("Retirement in Romania is at " + retirementAge);
        System.out.println("I have to work " + yearsBeforeRetirement + " years from now on");
    }
}